examples/platform_view/windows/flutter/CMakeLists.txt

```diff
@@ -1,7 +1,3 @@
-# Copyright 2014 The Flutter Authors. All rights reserved.
-# Use of this source code is governed by a BSD-style license that can be
-# found in the LICENSE file.
-
```

Comment:
Please keep the license header if it exists

Reply:
I just copied the template, and indeed, it's not fully coherent between all the copies of this file.